xgcm/test/test_metrics.py

```diff
@@ -311,28 +307,86 @@ def test_set_metric():
         assert k in grid_manual._metrics.keys()
 
         for metric_expected, metric in zip(v, grid_manual._metrics[k]):
-            xr.testing.assert_equal(metric_expected.reset_coords(drop=True), metric)
+            xr.testing.assert_allclose(metric_expected.reset_coords(drop=True), metric)
 
         for metric_expected, metric in zip(v, grid._metrics[k]):
-            xr.testing.assert_equal(metric_expected.reset_coords(drop=True), metric)
+            xr.testing.assert_allclose(metric_expected.reset_coords(drop=True), metric)
 
 
-@pytest.mark.skip(reason="For the next PR")
 @pytest.mark.parametrize(
-    "metric_axes,metric_name",
+    "metric_axes, metric_varname, expected_varname",
     [
-        ("X", "dx_t"),
-        ("Y", "dy_ne"),
+        ("X", "dx_n", ["dx_t", "dx_n_overwrite", "dx_e", "dx_ne"]),
+        (
+            ("Y", "X"),
+            "area_n",
+            ["area_t", "area_n_overwrite", "area_e", "area_ne"],
+        ),
     ],
 )
-def test_interp_metrics(metric_axes, metric_name):
-    # need to test with boundary and fill_value conditions
+def test_set_metric_overwrite_true(metric_axes, metric_varname, expected_varname):
+    ds, coords, metrics = datasets_grid_metric("C")
+
+    mv_modified = f"{metric_varname}_overwrite"
+    ds = ds.assign_coords({mv_modified: ds[metric_varname] * 10})
+
+    grid = Grid(ds, coords=coords, metrics=metrics)
+
+    grid.set_metrics(metric_axes, mv_modified, overwrite=True)
+
+    if len(metric_axes) > 1:
```

Comment:
Hacky approach to using `metric_axes = ("X","Y")`😅  
Feel free to suggest an alternative!

Reply:
does `frozenset(list(metric_axes))` work in both cases?

Reply:
It worked!